modules/data-catalog-tag-template/variables.tf

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "factory_config_path" {
+  description = "Path to data files and folders that enable factory functionality."
+  type        = string
```

Comment:
Can you align to this naming and type?

https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/modules/organization/variables.tf#L31

Reply:
done